python/hail/expr/functions.py

```diff
@@ -670,6 +670,33 @@ def parse_locus(s, reference_genome=None):
     return construct_expr(ApplyMethod('Locus({})'.format(reference_genome.name), s._ast), TLocus(reference_genome),
                           s._indices, s._aggregations, s._joins, s._refs)
 
+@typecheck(gp=expr_list)
+def gp_dosage(gp):
+    """
+    Return expected genotype dosage from array of genotype probabilities. Only defined for bi-allelic variants. The
```

Comment:
line too long!

Reply:
Split this out into a quick summary and a notes section that describes the calculation